ansible_collections/arista/avd/docs/contribution/setup-environment.md

```diff
@@ -244,11 +244,11 @@ Your environment is now ready and you can start to build your own project levera
 
 ## Using Python 3 Virtual Environment feature
 
-This section describes how to configure python to run ansible and AVD.
+This section describes how to configure python to run the ansible AVD.
```

Comment:
I liked it better the way it was phrased before. Possibly rephrase to "...to run ansible-avd" or simply "...to run AVD".